src/test/java/poll/service/VoteServiceTest.java

```diff
@@ -1,4 +1,78 @@
 package poll.service;
 
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import poll.repository.Answer;
+import poll.repository.Vote;
+import poll.repository.VoteRepository;
+
+import java.time.LocalDateTime;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.when;
+
+
+@ExtendWith(MockitoExtension.class)
 public class VoteServiceTest {
+
+    @Mock
+    VoteRepository voteRepository;
+
+    @InjectMocks
+    VoteService voteService;
+
+
+    @Test
+    public void should_ReturnVote_when_GivenVoteId() {
```

Comment:
The name is very good. What do you think about providing more information like: 
`should_ReturnVote_when_GivenVoteIdAndVoteFound`